qemu/tests/balloon_hotplug.py

```diff
@@ -70,6 +91,11 @@ def run_pm_test(pm_test, plug_type):
         vm.params["balloon_dev_devid"] = "balloon%d" % idx
         vm.params["balloon_dev_add_bus"] = "yes"
 
+        if params.get("blnsrv_enable", "no") == "yes":
```

Comment:
Do we need to run this test with `blnsrv_enable = no` on Windows? If no, we do not need this param, and the thing would be like:
```python
def enable_balloon_service():
    if params['os_type'] != 'windows':
        return
    error_context.context("Install and check balloon service in windows"
                          "guest", logging.info)
    session = vm.wait_for_login()
    ...


...
enable_balloon_service()
...
```